spec.md

```diff
@@ -880,9 +880,10 @@ message ControllerServiceCapability {
 
 #### `NodePublishVolume`
 
-A Node Plugin MUST implement this RPC call.
-This RPC is typically called by the CO when it wants to place a workload that wants to use the volume on a node.
+This RPC is called by the CO when a workload that wants to use the specified volume is placed (scheduled) on a node.
 The Plugin SHALL assume that this RPC will be executed on the node where the volume will be used.
+This RPC may be called by the CO multiple times: at least once per volume per workload
```

Comment:
I am not sure if we want to dictate that in the spec and callout "workload" specifically. Or we can just say, the CO MAY call `NodePublishVolume` multiple times on the node with different `target_path`. It's up to CO to decide whether to call this RPC multiple times or not.

Also, I think we need to call out the reverse workflow in the spec. More specifically, when to call `ControllerUnpublishVolume` if `NodeUnpublishVolume` is called multiple times? One idea is to say: `ControllerUnpublishVolume` MUST be called after `NodeUnpublishVolume` is called for each `target_path` and succeeds.